pages/common/miniserve.md

```diff
@@ -0,0 +1,17 @@
+# miniserve
+
+> Simple HTTP file server CLI.
+> For when you really just want to serve some files over HTTP right now!.
```

Comment:
I personally, don't think this is necessary.

Reply:
Should I delete this line?

Reply:
Hey there! Yeah, I think this extra line isn't really necessary :-)